src/applications/_mock-form-ae-design-patterns/mocks/endpoints/user/index.js

```diff
@@ -322,6 +322,20 @@ const baseUserResponses = {
           isVeteran: true,
           servedInMilitary: true,
         },
+        'view:maritalStatus': {
+          maritalStatus: 'Married',
+        },
+        spouseFullName: { first: 'Michelle', last: 'Smith' },
+        spouseDateOfBirth: '1990-08-03',
+        spouseSocialSecurityNumber: '451906574',
+        'view:reportDependents': true,
+        'view:skipDependentInfo': false,
+        dependents: [
+          {
+            fullName: { first: 'Mia', last: 'Smith' },
+            dependentRelation: 'Daughter',
+          },
+        ],
```

Comment:
Does this need to be in the user response? I thought this was something that we put in the in-progress-forms response?

Reply:
deleted